test/utils/helpers.ts

```diff
@@ -0,0 +1,3 @@
+export function testComponent(componentId: string) {
```

Comment:
```suggestion
export function testComponent(componentId: string): string {
```

ooh we're missing the return type on this, I'd need to fire this up in vs-code with my typescript tooling to check, but I think it just needs this. It's always gonna return a string, right?

Reply:
Thanks for this @huwd and yes it should always return a string indeed